include/RAJA/DepGraphNode.hxx

```diff
@@ -84,43 +86,59 @@ public:
    /// algorithms and platforms. We haven't determined the best defaults yet!
    ///
    static const int _MaxDepTasks_          = 8;
-   static const int _SemaphoreValueAlign_  = 256;
 
    ///
    /// Default ctor initializes node to default state.
    ///
    DepGraphNode()
       : m_num_dep_tasks(0),
         m_semaphore_reload_value(0),
-        m_semaphore_value(0) 
-   { 
-      posix_memalign((void **)(&m_semaphore_value), 
-                     _SemaphoreValueAlign_, sizeof(int)) ;
-      *m_semaphore_value = 0 ;       
-   }
-
-   ///
-   /// Dependency graph node dtor.
-   ///
-   ~DepGraphNode() { if (m_semaphore_value) free(m_semaphore_value); }
+        m_semaphore_value(0) { }
```

Comment:
The purpose of the alignment of m_semaphore_value in the old version was an attempt to prevent thrashing. Since each core may own it's own semaphore, we did not want multiple semaphores to accidentally fall in the same 'coherence page', causing the page to move around among owning cores as any of the semaphores are updated.